packages/backend-proxy-middleware/src/base/config.ts

```diff
@@ -1,44 +1,19 @@
 /**
- * Get the effective proxy string from runtime args (highest priority), given config value or environment variables.
+ * Updates the proxy configuration with values from runtime args (highest priority), given config value or environment variables.
  *
  * @param proxyFromConfig - optional proxy string from configuration
- * @returns proxy server if required, otherwise undefined
  */
-export function getCorporateProxyServer(proxyFromConfig?: string): string | undefined {
+export function updateProxyEnv(proxyFromConfig?: string): void {
     let proxyFromArgs: string | undefined;
     process.argv.forEach((arg) => {
         if (arg.match(/proxy=/g)) {
             proxyFromArgs = arg.split('=')[1];
         }
     });
-    return (
-        proxyFromArgs ||
-        proxyFromConfig ||
-        process.env.FIORI_TOOLS_PROXY ||
-        process.env.http_proxy ||
-        process.env.HTTP_PROXY ||
-        process.env.https_proxy ||
-        process.env.HTTPS_PROXY ||
-        process.env.npm_config_proxy ||
-        process.env.npm_config_https_proxy
-    );
-}
+    const proxyFromFioriToolsConfig = proxyFromArgs || proxyFromConfig || process.env.FIORI_TOOLS_PROXY;
```

Comment:
I wonder whether this causes that values set in `process.env` are overwritten by config from `ui5.yaml`.